packages/idyll-cli/src/index.js

```diff
@@ -1,330 +1,397 @@
-const fs = require('fs');
```

Comment:
Big diff for this file - is this a full re-write or just Git being annoying? If you can point to the parts that changed it will be helpful

Reply:
Is the change to `searchParentDirectories` the only existing functionality that changed?

Reply:
Yes that and then the new functionality. Will clean up based on your other comment about that one liner

Reply:
Just pushed a merged version of this. I think Git was just being funny, I handled the merge by copying idyll master over and making the edits from my fork into that so I didn't miss anything. Should be relatively straightforward to see the line changes but the diff is ugly